Simplified/AxisService/AxisService.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  AxisService.swift
+//  Simplified
+//
+//  Created by Raman Singh on 2021-04-01.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+@objc
+protocol BookDownloadBroadcasting {
```

Comment:
all our protocols and classes should have the NYPL prefix, despite the swift convention)

Reply:
Fixed.